src/components/Search/Search.tsx

```diff
@@ -32,7 +34,11 @@ const ActiveBox = styled(Box)`
     }
 `;
 
-const SearchInputContainer = styled(Box)`
```

Comment:
I'm not quite getting why this change is here

Reply:
This changed is needed after removing the `StyledComponentBase` typing from `Box`. As `isInFocus` is not a prop in `BoxProps`, the interface is extended here.